test/LANDRegistry.js

```diff
@@ -387,6 +387,17 @@ contract('LANDRegistry', accounts => {
         await land.initialize(hacker, { from: hacker })
         await assertRevert(land.forbidDeploy(hacker, { from: hacker }))
       })
+
+      it('throw if user tries to assign LAND and it not deployer', async function() {
```

Comment:
I think it is better to put `it('reverts....`

Reply:
👍